adhocracy-plus/templates/account/email.html

```diff
@@ -8,51 +8,64 @@
 {% block dashboard_content %}
 <h1 class="mt-0">{% trans 'Email Addresses' %}</h1>
 
-<form class="multiform" action="{% url 'account_email' %}?next={{ request.path|urlencode }}" method="post">
-    {% csrf_token %}
+<section>
+    <h2>{% trans 'Main Email' %}</h2>
+    <span>{{ user.email }}</span>
+    <span class="label">{% trans "Primary" %}</span>
+    <span class="label">{% trans "Verified" %}</span>
+</section>
+
+{% if user.emailaddress_set.all.count > 1 %}
+<section>
+    <h2 class="bg--top-border">{% trans 'Other Emails' %}</h2>
     {% for emailaddress in user.emailaddress_set.all %}
-    <div class="form-check">
-        <label class="form-check__label">
-            <input type="radio" name="email"
-                   {% if emailaddress.primary or user.emailaddress_set.count == 1 %}checked="checked" {% endif %}
-                   value="{{emailaddress.email}}">
-            {{ emailaddress.email }}
-            {% if emailaddress.primary %}
-            <span class="label">{% trans "Primary" %}</span>
-            {% endif %}
-            {% if emailaddress.verified %}
-            <span class="label">{% trans "Verified" %}</span>
-            {% else %}
-            <span class="label">{% trans "Unverified" %}</span>
-            {% endif %}
-        </label>
-    </div>
+        {% if not emailaddress.primary %}
+            <form class="multiform" action="{% url 'account_email' %}?next={{ request.path|urlencode }}" method="post">
+                {% csrf_token %}
+                <input type="hidden" name="email"
+                       value="{{emailaddress.email}}" >
+                {{ emailaddress.email }}
+                {% if emailaddress.verified %}
+                <span class="label mb-3">{% trans "Verified" %}</span>
+                {% else %}
+                <span class="label mb-3">{% trans "Unverified" %}</span>
+                {% endif %}
+                <ul class="u-list-reset">
+                {% if not emailaddress.primary and emailaddress.verified %}
```

Comment:
No it is not necessary, but I wanted to be sure. Because the other emailaddress is not from the list, but from the user's email. That should be the only primary address. Also, if we remove the condition above to not show the primary one this would fail. Soo, let's keep it in?